Mathlib/Algebra/BigOperators/Finsupp.lean

```diff
@@ -564,47 +600,47 @@ theorem Finset.sum_apply' : (∑ k in s, f k) i = ∑ k in s, f k i :=
   (Finsupp.applyAddHom i : (ι →₀ A) →+ A).map_sum f s
 #align finset.sum_apply' Finset.sum_apply'
 
-theorem Finsupp.sum_apply' : g.Sum k x = g.Sum fun i b => k i b x :=
+theorem Finsupp.sum_apply' : g.sum k x = g.sum fun i b => k i b x :=
   Finset.sum_apply _ _ _
 #align finsupp.sum_apply' Finsupp.sum_apply'
 
 section
 
-include h0 h1
+-- Porting note: removed the following, expected command
+-- include h0 h1
 
 open Classical
 
-theorem Finsupp.sum_sum_index' : (∑ x in s, f x).Sum t = ∑ x in s, (f x).Sum t :=
+theorem Finsupp.sum_sum_index' : (∑ x in s, f x).sum t = ∑ x in s, (f x).sum t :=
   Finset.induction_on s rfl fun a s has ih => by
     simp_rw [Finset.sum_insert has, Finsupp.sum_add_index' h0 h1, ih]
 #align finsupp.sum_sum_index' Finsupp.sum_sum_index'
 
-end
+-- Porting note: unknown namespace end
+-- end
```

Comment:
This is a bit weird. I would expect this to end the section started on L607.

Reply:
There used to be an error here but I don't see it anymore. Maybe hallucinations. Thanks!